FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureStorageJobFactory.cs

```diff
@@ -114,15 +114,10 @@ private IJob CreateOrchestratorTask(JobInfo jobInfo)
                     // return null if the job version is unsupported
                     if (isSupported)
```

Comment:
We can set a new version for splitter? So the old version orchestrator job will work in the old way and we don't need to care about the compatibility of splitter with the old jobs.

Reply:
Set version as V3 and support old versions, thanks.